forkpi/spoonpi/spoonpi/forkpi_db.py

```diff
@@ -33,6 +43,16 @@ def authenticate(self, *args, **kwargs):
              This is an empty list if is_authorized is False
         """
         keywords = kwargs.keys()
+
+        if AuthKeys.GLOBAL_PIN.value in keywords:
+            global_pin = kwargs[AuthKeys.GLOBAL_PIN.value]
+            user = User.objects.get(username=kwargs.get("username"))
+            if global_pin == AppConfig.objects.get_or_update_global_pin():
```

Comment:
@umairwaheed its not making sense to me, we should not create global pin here, the global pin should be created by admins or superusers only

Reply:
@Shrhawk yes, we are just getting the pin here, not creating it.